### PR TITLE
Day4 Challenge01

### LINKED ISSUE #259: Challenge 1: Create a GitHub project and add some cards which reflects the work for day4.
e.g.
- Import the Azure Dev College's training repository to your org 
- Create your first Pull Request
- Deploy the sample application to your subscription

### PR DESCRIPTION
**Pull request by AndreasM009**
closes #259

### CODE REVIEW

File: day4/challenges/01-challenge-boards.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+![GitHub create project part2](./images/gh-create-project-part2.png)
+
+After the project was created, we can add notes to plan and track our work. There are already some example notes created in the _To Do_ column of the board. You can delete these notes, as we don't need them. A project board is made up of issues, pull requests and notes which are displayed as cards. Cards are categorized in columns of your choice. We can move cards from one column to another. As we have selected the _Automated kanban_ template, the board is made up of three columns, but you can add more columns to the board to categorize cards the way you want it. 
+
+The project is already linked to the repository _myfirst-repo_. There is a _Menu_ button at the upper right corner of the board. When you click the button, you get an overview of activities and a list of linked repositories. 
+
+:::tip
+📝 _Maybe the button is not visible, because there is the flyout dislayed that allows you to add cards. Close it and you will see the Menu button._
```
Comment (waeltken) [inline day4/challenges/01-challenge-boards.md:48]: Go easy on italics here... it's already a textbox.

File: day4/challenges/01-challenge-boards.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+
+At the moment ther project creator, organisation owners and members are the only one who can add and manage cards. We need to assign permissions to a team so they can create and manage cards.
+Open the board's menu again. Click the ellipsis _..._ button and open the _Settings_. 
+
+![GitHub board menu ellipses](./images/gh-board-menu-ellipses.png)
+
+In the _Options_ section you can manage the visibility of the project and organization member's permissions. _Write_ is the default baseline permission level for all members of the organization. 
+
+:::tip
+📝 _To restrict access to all organisation members, set Read as baseline permission level._
```
Comment (waeltken) [inline day4/challenges/01-challenge-boards.md:63]: Go easy on italics here... it's already a textbox.

File: day4/challenges/01-challenge-boards.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+
+![GitHub issue details](./images/gh-issue-details.png)
+
+Your board should now look like this:
+
+![GitHub board overview-01](./images/gh-board-overview-01.png)
+
+Now you may wonder why you convert a note into an issue. In an issue we can additionally specify more concrete information. We can specify who is responsible for the issue, which repository it refers to and we can assign labels to classify the issue. With this approach, we could map the following workflow: 
+
+_We use notes to plan and prioritize the work for our next milestone (sprint). When we start working on the task, we convert the note into an issue, choose a repository, assign a responsible person, classify it and push it to the _In progress_ column._
```
Comment (waeltken) [inline day4/challenges/01-challenge-boards.md:124]: Paragraph in all italics: maybe use :::tip?

File: day4/challenges/01-challenge-boards.md (hunk 1 above)
Comment (AndreasM009) [reply day4/challenges/01-challenge-boards.md:48]: Yes you are right. We should change this in our templates, too.
https://github.com/azuredevcollege/trainingdays/blob/master/templates/dayX/challenges/NN-challenge-bo.md

File: day4/challenges/01-challenge-boards.md (hunk 2 above)
Comment (AndreasM009) [reply day4/challenges/01-challenge-boards.md:63]: Yes you are right. We should change this in our templates, too.
https://github.com/azuredevcollege/trainingdays/blob/master/templates/dayX/challenges/NN-challenge-bo.md

**Review by waeltken — APPROVED** [review]

**Pull request merged**